server/src/main/java/org/apache/druid/metadata/SQLMetadataStorageActionHandler.java

```diff
@@ -82,7 +83,8 @@ public SQLMetadataStorageActionHandler(
   )
   {
     this.connector = connector;
-    this.jsonMapper = jsonMapper;
+    //noinspection UnnecessaryFullyQualifiedName
+    this.jsonMapper = new DefaultObjectMapper((DefaultObjectMapper) jsonMapper).addMixIn(org.apache.druid.metadata.PasswordProvider.class, org.apache.druid.metadata.PasswordProviderRedactionMixIn.class);
```

Comment:
Same here. Would you please explain why this change is needed? `KinesisIndexTask` is already using `PasswordProvider`, so it should work without this change.